docs/en_US/Compressor/WeightRankFilterPruner.md

```diff
@@ -37,7 +63,7 @@ pruner.compress()
 - **sparsity:** This is to specify the sparsity operations to be compressed to
 - **op_types:** Only Conv2d is supported in L1Filter Pruner
 
-## 3. Experiment
+## 4. Experiment
```

Comment:
there are three pruning algorithms above, it is not clear which one produces the following experiment results.

Reply:
resolved